tests/src/system/basic/WskBasicTests.scala

```diff
@@ -339,24 +339,6 @@ class WskBasicTests
             }
     }
 
-    /**
-     * Tests creating an nodejs action that throws a whisk.error() response. The error message thrown by the
-     * whisk.error() should be returned.
-     */
-    it should "create and invoke a blocking action resulting in a whisk.error response" in withAssetCleaner(wskprops) {
```

Comment:
The tests are to confirm returning {error:...} work as expected with the cli for example.